homeassistant/components/eheimdigital/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
```

Comment:
We're still new to this. But I would suggest for the initial PR we should just only add the bronze rules. Even though its the same code, I notice that the way I review code is different when I am checking for a new integration or when I am checking if the scale is applied. At this moment I feel like that there is a large overhead in checking for both the integration being good and checking if it satisfies the tiers above bronze.

Maybe in the future when a lot of the checks are automated, this could be better.

Also feel free to let me know what you think about this by the way.

Can you also add the base checklist to the PR description and add the quality_scale field to the manifest.json?